app/src/main/java/org/oppia/app/topic/revisioncard/RevisionCardFragmentPresenter.kt

```diff
@@ -27,6 +31,10 @@ class RevisionCardFragmentPresenter @Inject constructor(
 
     viewModel.setSubtopicIdAndBinding(topicId, subtopicId, binding)
 
+    binding.revisionCardReturnButton.setOnClickListener{
```

Comment:
Click listener should be managed from `ViewModel` similar to  how it is implemented in `OngoingStoryViewModel`

Reply:
Done